1-exercises/E-for-loop/exercise1.js

```diff
@@ -7,8 +7,8 @@
 
 // Change the below code to use a for loop instead of a while loop.
 let i = 0;
-while(i < 26) {
+for(i = 0; i < 26; i++)
```

Comment:
Nice attempt. But with a for loop, we initialise the "i" variable in the for loop brackets. What change needs to be done to correct this? Take our Line 9 and add the let before "i=" in the current line 10 perhaps